virttest/guest_agent.py

```diff
@@ -581,6 +581,14 @@ def set_time(self, nanoseconds=None):
         return self.cmd(cmd=cmd, args=args)
 
     @error_context.context_aware
+    def guest_info(self):
```

Comment:
In check-memory-leak test case, just need to repeat send cmd "guest-info", no need to get supported cmds list. And get supported cmds covered by other test case.